Makefile

```diff
@@ -6,4 +6,4 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && npm test"
+	docker run --rm -v $$PWD/test/fixtures:/code $(IMAGE_NAME) sh -c "cd /usr/src/app && npm test"
```

Comment:
Yes, I thought that's how you get a variable to pass through. Seems to work? :dancers: